Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -200,15 +203,18 @@ def __find_groups(self) -> Tuple[Dict[str, int], Set[Tuple[int, int]]]:
                 info["box"] += 1
                 group_blocks.update(box_blocks)
 
-        return info, group_blocks
+        return info, list(group_blocks)
```

Comment:
```suggestion
        return info, group_blocks
```

Reply:
In line 167, we are using the returned value from `__find_groups` to be the parameter for `remove_blocks`, which requires a list of coordinates.